test/EmptyApp.FunctionalTests/Startup.cs

```diff
@@ -31,7 +35,7 @@ public void ConfigureServices(IServiceCollection services)
         public void Configure(IApplicationBuilder app)
         {
             app.UseApplicationInsightsRequestTelemetry();
-            app.UseDeveloperExceptionPage(new ErrorPageOptions());
+            app.UseDeveloperExceptionPage(new DeveloperExceptionPageOptions());
```

Comment:
Probably don't need to pass any param to this at all if you just want the default settings.